*** raspi-config.orig.txt	RPi-Distro/raspi-config pios/20261026 (trixie)
--- raspi-config.rqb2.txt	RasQberry
***************
*** 15,20 ****
--- 15,29 ----
  fi
  CONFIG=/boot${FIRMWARE}/config.txt
  
+ # modification for RasQberry
+ # Load RQB2 menu from global config location
+ RQB2_menu="/usr/config/RQB2_menu.sh"
+ if [ -e "$RQB2_menu" ] ; then
+   . "$RQB2_menu"
+ else
+   echo "ERROR: $RQB2_menu does not exist !"
+ fi
+ 
  USER=${SUDO_USER:-$(who -m | awk '{ print $1 }')}
  if [ -z "$USER" ] && [ -n "$HOME" ]; then
    USER=$(getent passwd | awk -F: "\$6 == \"$HOME\" {print \$1}")
***************
*** 4240,4245 ****
--- 4249,4255 ----
          FMEMSIZE="$(expr $MEMSIZE / 1024)GB"
        fi
        FUN=$(whiptail --title "Raspberry Pi Software Configuration Tool (raspi-config)" --backtitle "$(cat /proc/device-tree/model), ${FMEMSIZE}" --menu "Setup Options" $WT_HEIGHT $WT_WIDTH $WT_MENU_HEIGHT --cancel-button Finish --ok-button Select \
+         "0 RasQberry" "Quantum demos, LED panel and settings" \
          "1 System Options" "Configure system settings" \
          "2 Display Options" "Configure display settings" \
          "3 Interface Options" "Configure connections to peripherals" \
***************
*** 4265,4270 ****
--- 4275,4282 ----
        do_finish
      elif [ $RET -eq 0 ]; then
        case "$FUN" in
+         0\ *) do_rasqberry_menu ;;
+         9\ *) do_rasqberry_about ;;
          1\ *) do_system_menu ;;
          2\ *) do_display_menu ;;
          3\ *) do_interface_menu ;;
